test/integration/scheduler_perf/scheduler_bench_test.go

```diff
@@ -31,22 +34,80 @@ import (
 // BenchmarkScheduling benchmarks the scheduling rate when the cluster has
 // various quantities of nodes and scheduled pods.
 func BenchmarkScheduling(b *testing.B) {
-	tests := []struct{ nodes, pods int }{
-		{nodes: 100, pods: 0},
-		{nodes: 100, pods: 1000},
-		{nodes: 1000, pods: 0},
-		{nodes: 1000, pods: 1000},
+	tests := []struct{ nodes, pods, minOps int }{
+		{nodes: 100, pods: 0, minOps: 100},
```

Comment:
Please rename `pods` to `existingPods` and `minOps` to `minPods`. Also, please add a comment explaining that existingPods are created before the measurements start and then at least `minPods` are created.

Reply:
Done